packages/react-composites/review/react-composites.api.md

```diff
@@ -525,6 +525,7 @@ export interface CallWithChatCompositeProps extends BaseCompositeProps<CallWithC
 
 // @public
 export interface CallWithChatCompositeStrings {
+    chatButtonEnhancedToolTipContent: string;
```

Comment:
This name will confuse a Contoso dev trying to override strings (enhanced in what way?)

How about, `chatButtonMessageCountToolTipContent`

(aside: I don't like the "Content" suffix in all these names. All strings are "contents", we could just say `chatButtonMessageCountToolTip` and it would convey the same thing. I know we're just following Fluent's naming here, so it's good to stick with the longer name)

Reply:
okay will change these strings so that the names are more appropriate